admin-dev/themes/new-theme/js/components/modal/confirm-modal.ts

```diff
@@ -0,0 +1,184 @@
+/**
```

Comment:
You're already in modal directory, could you please remove the `-modal` in the filename?

Reply:
That makes sense, but I don't know I prefer to keep the filename and the main component in it in "sync"

Reply:
As you wish, I'm not in favor of duplicating types in the filename when the scope is already defined by its parent directory :sweat_smile: